client/starwhale/base/scheduler/task.py

```diff
@@ -32,13 +32,15 @@ def __init__(
         context: Context,
         workdir: Path,
         step: Step,
+        handlerargs: t.List[str] = [],
```

Comment:
`[]` should not be as the default value.

Reply:
Thank you for pointing this out